src/Microsoft.Diagnostics.Monitoring.WebApi/Models/EgressOperationStatus.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+#if UNITTEST
+namespace Microsoft.Diagnostics.Monitoring.UnitTests.Models
+#else
+namespace Microsoft.Diagnostics.Monitoring.WebApi.Models
+#endif
+{
+    /// <summary>
+    /// Represents a partial model when enumerating all operations.
+    /// </summary>
+    public class OperationSummary
+    {
+        public Guid Id { get; set; }
+
+        public DateTime CreatedDateTime { get; set; }
+
+        public OperationState Status { get; set; }
+    }
+
+    /// <summary>
+    /// Represents the state of a long running operation. Used for all types of results, including
+    /// successes and failures.
+    /// </summary>
+    public class OperationStatus
```

Comment:
I don't fully understand the relationship, so it might not be appropriate, but would it make sense for OperationStatus to inherit from OperationSummary (since the summary should always be a subset of the full status)? The duplication of the getters and setters between the classes caught my eye, so I thought I'd mention it. Also, relatedly, am I correct in assuming Id is the same as OperationId? In which case I'd think they should probably be named the same, and if not maybe a comment to point out the difference?

Reply:
I thought about doing this but I really wanted to enforce that a new object had to be made here. I'll correct the names and add some comments.